packages/react-scripts/utils/createJestConfig.js

```diff
@@ -20,8 +20,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     moduleNameMapper: {
-      '^.+\\.(?!(js|jsx|css|json)$)[^\\.]+$': resolve('config/jest/FileStub.js'),
```

Comment:
Yeah, I guess I should get rid of the lone `.css` really because it's actually redundant, so likely to just cause confusion?